frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/patients-tab/simulated-region-overview-patients-tab/simulated-region-overview-patients-tab.component.ts

```diff
@@ -0,0 +1,61 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { createSelector, Store } from '@ngrx/store';
+import type { PatientStatus } from 'digital-fuesim-manv-shared';
+import { Patient, SimulatedRegion, UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    createSelectElementsInSimulatedRegion,
+    selectConfiguration,
+    selectPatients,
+} from 'src/app/state/application/selectors/exercise.selectors';
+import { SelectPatientService } from '../../../select-patient.service';
+import { comparePatientsByVisibleStatus } from '../../compare-patients';
+
+@Component({
+    selector: 'app-simulated-region-overview-patients-tab',
+    templateUrl: './simulated-region-overview-patients-tab.component.html',
+    styleUrls: ['./simulated-region-overview-patients-tab.component.scss'],
+})
+export class SimulatedRegionOverviewPatientsTabComponent implements OnInit {
+    @Input() simulatedRegion!: SimulatedRegion;
+    @Input() selectedPatientId?: UUID;
+
+    patients$!: Observable<(Patient & { visibleStatus: PatientStatus })[]>;
+    selectedPatientVisibleStatus$?: Observable<PatientStatus>;
```

Comment:
Where is this attribute needed?

Reply:
This isn't needed in the most recent version, I've removed it in https://github.com/hpi-sam/digital-fuesim-manv/pull/739/commits/38107cf4a5babbd954641f713612a8d8f2c9f4cb